examples/unitTestExample/README.md

```diff
@@ -0,0 +1,4 @@
+# Setup
+
+Tests can be run via the command line without fail, however, to get them to run in Android Studio you will need to perform the following steps in your run configurations in Android Studio:
```

Comment:
Also add which commands to run to execute the tests. `gradle connectedCheck` and `gradle test` I assume?